Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -136,36 +138,41 @@ void parseMCSParametersJSON(const char* json, MCSParameters* params) {
 }
 
 MCSResult findMCS(const std::vector<ROMOL_SPTR>& mols,
-                  const MCSParameters* params) {
+                  const MCSParameters* params,
+                  void* userData) {
   MCSParameters p;
   if (nullptr == params) {
     params = &p;
   }
-  RDKit::FMCS::MaximumCommonSubgraph fmcs(params);
+  RDKit::FMCS::MaximumCommonSubgraph fmcs(params, userData);
   return fmcs.find(mols);
 }
 
 MCSResult findMCS_P(const std::vector<ROMOL_SPTR>& mols,
-                    const char* params_json) {
+                    const char* params_json,
+                    void* userData) {
   MCSParameters p;
   parseMCSParametersJSON(params_json, &p);
-  return findMCS(mols, &p);
+  return findMCS(mols, &p, userData);
 }
 
 MCSResult findMCS(const std::vector<ROMOL_SPTR>& mols, bool maximizeBonds,
                   double threshold, unsigned timeout, bool verbose,
                   bool matchValences, bool ringMatchesRingOnly,
                   bool completeRingsOnly, bool matchChiralTag,
+                  float maxDistance, const std::vector<unsigned> conformerIdxs,
```

Comment:
Should I remove these options from the argument list and have them available only through the parameter object?